apps/console/src/features/roles/components/wizard-updated/role-basics.tsx

```diff
@@ -0,0 +1,388 @@
+                                value={ audience }
+                                defaultValue={ initialValues?.roleAudience ?? RoleConstants.DEFAULT_ROLE_AUDIENCE }
+                                data-componentid={ `${componentId}-${audience}-audience` }
+                                listen={ () => setRoleAudience(audience) }
+                                hint={ 
+                                    index === Object.keys(RoleAudiences).length- 1 
+                                        ? (
+                                            <Trans 
+                                                i18nKey= { "console:manage.features.roles.addRoleWizard.forms." + 
+                                                    "roleBasicDetails.roleAudience.hint" }>
+                                                Set the audience of the role.
+                                                <b>Note that audience of the role cannot be changed.</b>
+                                            </Trans>
+                                        ) 
+                                        : null 
+                                        // TODO: need to add a learn more for this.
+                                }
+                            />
+                        ))
+                }
+            </div>
+            {
+                !isDisplayNoAppScopeApplicatioError
+                    ? (
+                        <Alert severity="info">
+                            {
+                                roleAudience === RoleAudiences.ORG
+                                    ? t("console:manage.features.roles.addRoleWizard.forms.roleBasicDetails.notes" + 
+                                        ".orgNote")
+                                    : t("console:manage.features.roles.addRoleWizard.forms.roleBasicDetails.notes" + 
+                                        ".appNote")
+                                // TODO: need to add a learn more for this.
+                            }
+                        </Alert>
+                    ) : (
+                        <Alert severity="error">
+                            <Trans 
+                                i18nKey= { "console:manage.features.roles.addRoleWizard.forms.roleBasicDetails.notes" + 
+                                    ".cannotCreateRole" }
+                            >
+                            You cannot create an application-scoped role because there are currently no applications 
+                            that support application-scoped role. Please (
+                                <Link
+                                    data-componentid={ `${componentId}-link-api-resource-page` }
+                                    onClick={ navigateToApplications }
+                                    external={ false }
+                                >
+                                    create an application
```

Comment:
Is this i18n'ed?

Reply:
Yes I have added the localization for this.